src/main/java/org/example/homework_nr_8/Animal.java

```diff
@@ -0,0 +1,24 @@
+package org.example.homework_nr_8;
+
+public interface Animal {
+
+    Integer PairOfLegs = 2; //Статическая переменная. Количество ног в одной паре.
```

Comment:
interface final variables must follow the correct name convention. In this case PAIR_OF_LEGS = 2;